tests/instancetype/instancetype.go

```diff
@@ -47,6 +47,19 @@ import (
 
 var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-compute] Instancetype and Preferences", decorators.SigCompute, func() {
 	var virtClient kubecli.KubevirtClient
+	const (
+		preferredTerminationGracePeriodSeconds = 15
+		expectedCausesLength                   = 3
+		timeout                                = 300 * time.Second
+		dvSuccessTimeoutSeconds                = 180
+		guestMemorySize                        = 2
+		numCore                                = 2
+		numSocket                              = 2
+		numThread                              = 2
+		MemorySize4                            = 4
+		memorySize2GB                          = 2 * 1024 * 1024 * 1024
+		memorySize1GB                          = 1 * 1024 * 1024 * 1024
+	)
```

Comment:
Are all these constants really necessary? I would rather only add constants for the places that gomnd is complaining about.

Reply:
@0xFelix  ok three constant are added additionally , I'll remove that